openstack/rds/v3/configurations/results.go

```diff
@@ -78,14 +112,42 @@ func (r GetResult) Extract() (*Configuration, error) {
 	return &response, err
 }
 
+// Extract is a function that accepts a result and extracts a list of configurations.
+func (lr ListResult) Extract() ([]Configuration, error) {
```

Comment:
The reciever is named `r` normally. See [naming](https://github.com/opentelekomcloud/gophertelekomcloud/blob/master/STYLEGUIDE.md#naming) guide